python/eups/tags.py

```diff
@@ -452,16 +452,7 @@ def parse(name, defGroup=Tags.global_):
         there is not group name, it is assumed to be of the group given by
         defGroup (which defaults to global).
         """
-        # This is requires python 2.4
-        #   parts = name.rsplit(':', 1)
-        # replaced with:
-        parts = name.split(':')
-        if len(parts) > 2:
-            try:
-                parts = ":".join(parts[:-1], parts[-1])
```

Comment:
This must also not have ever been called, as you can't call `join` like this.  I think it was meant to be `[":".join(parts[:-1]), parts[-1]]`.